scroll-animations-1/Overview.bs

```diff
@@ -825,78 +732,66 @@ spec: cssom-view-1; type: dfn;
-			[=attaches=] to the corresponding [=view progress timeline=]
-			defined on this box.
+# Deferred Progress Timelines # {#deferred-timeline}
 
-		<dt><dfn>defer</dfn>
-		<dd>
-			This [=view progress timeline=] name’s [=attachment=]
-			is deferred to its descendants.
-			In other words,
-			this element declares a [=view progress timeline=]’s name
-			and defines its scope (see [[#timeline-scope]]),
-			but defers the timeline’s definition
-			([=view progress subject|subject=],
-			 [=scroll container=] source,
-			 axis, and
-			 [=view progress visibility range|visibility range=])
-			to its descendants.
-
-			If more than one matching descendant [=view progress timeline=] exists,
-			or if no such timeline exists,
-			then this timeline name has no [=attachment=],
-			and represents an [=inactive timeline=].
-
-		<dt><dfn>ancestor</dfn>
-		<dd>
-			This [=view progress timeline=] definition
-			[=attaches=] to the closest matching [=view progress timeline=] name
-			that was deferred (''view-timeline-name/defer'')
-			by an ancestor (if any).
+<dfn export>Deferred progress timelines</dfn> are timelines linked to the progress
+of another timeline found among the descendant elements.
+This effectively allows a [=scroll progress timeline=] or [=view progress timeline=]
+defined on a given element
+to expand its [[#timeline-scope|scope]] beyond that element and its descendants.
 
-			If no such ancestor exists,
-			the [=attachment=] is treated as ''view-timeline-attachment/local''.
-	</dl>
+[=Deferred progress timelines=] can be referenced in 'animation-timeline'
+by name (see [[#timeline-name-scope]])
+after declaring them using the 'timeline-scope' property.
 
-### View Timeline Shorthand: the 'view-timeline' shorthand ### {#view-timeline-shorthand}
+## Deferred Progress Timeline Ranges ## {#deferred-timelines-ranges}
 
-	<pre class='propdef shorthand'>
-	Name: view-timeline
-	Value: [ <<'view-timeline-name'>> [ <<'view-timeline-axis'>> || <<'view-timeline-attachment'>> ]? ]#
+A [=deferred progress timeline=] defines the same [=named timeline ranges=]
+its [=attached timeline=] defines.
+
+## Calculating Progress for a Deferred Progress Timeline ## {#deferred-timeline-progress}
+
+	Progress (the [=timeline/current time=]) for a [=deferred progress timeline=]
+	is the progress of its [=attached timeline=].
+	If there is no [=attached timeline=],
+	the timeline is [=inactive timeline|inactive=].
+
+	The <dfn export>attached timeline</dfn> of a [=deferred progress timeline=]
+	is the [=named scroll progress timeline=] or [=named view progress timeline=]
+	among the inclusive flat-tree descendants
+	with a matching name.
+	If more than one matching timeline exists,
+	or if no such timeline exists,
+	then the deferred timeline has no [=attached timeline=].
+
+## Naming a Deferred Progress Timeline: the 'timeline-scope' property ## {#timeline-scope}
+
+	<pre class='propdef'>
+	Name: timeline-scope
+	Value: none | [ <<dashed-ident>> ]#
```

Comment:
This is partially mixing up the dashed-ident change into the timeline-scope change, which leaves the spec in a weird inconsistent state.